numpy/lib/tests/test_shape_base.py

```diff
@@ -289,14 +289,22 @@ def test_functionality(self):
             assert_(b.shape[axis] == 1)
             assert_(np.squeeze(b).shape == s)
 
-    def test_deprecations(self):
-        # 2017-05-17, 1.13.0
+    def test_axis_tuple(self):
+        a = np.empty((3, 3, 3))
+        assert np.expand_dims(a, axis=(0, 1, 2)).shape == (1, 1, 1, 3, 3, 3)
+        assert np.expand_dims(a, axis=(0, -1, -2)).shape == (1, 3, 3, 3, 1, 1)
+        assert np.expand_dims(a, axis=(0, 3, 5)).shape == (1, 3, 3, 1, 3, 1)
+        assert np.expand_dims(a, axis=(0, -3, -5)).shape == (1, 1, 3, 1, 3, 3)
```

Comment:
Please add a test for the same axis being repeated. Is that supported?

Reply:
Repeated axes are not allowed.  `axis` refers to the output dimensions.

Reply:
Then please add a test showing that it is not allowed

Reply:
I added that test, thanks.